########################################
# BIG WHEELS (r = 0.085 m)
# 0.4587 RPS / 1000 pps
# 0.245 m/s / 1000 pps (scale_speed)
# 1 m/s -> 4081
# 5 m/s -> 20408
# max scale_steer: 0.23
# ######################################
# SMALL WHEELS (r = 0.055 m)
# 0.4896 RPS / 1000 pps
# 0.17 m/s / 1000 pps
# 1 m/s -> 5882
# 5 m/s -> 29412
# max scale_steer: 0.3 (overall max is 0.3)
# max scale_steer_turbo: 0.23 close to tip over but not quite 
########################################

axis_speed: 1  # Left thumb stick vertical
scale_speed: 5882
scale_speed_turbo: 29412

axis_steer: 3  # Right thumb stick horizontal
scale_steer: -0.3
scale_steer_turbo: -0.23

enable_button: 4  # Left bumper button
enable_turbo_button: 5  # Right bumper button

# BUTTONS
# A button........ button 0
# B button........ button 1
# X button........ button 2
# Y button........ button 3
# left bumper..... button 4
# right bumper.... button 5
# select button... button 6
# start button.... button 7
# xbox button..... button 8
# left stick...... button 9
# right stick..... button 10
# left D-pad...... button 11
# right D-pad..... button 12
# up D-pad........ button 13
# down D-pad...... button 14
# AXES
# left stick horizontal.... axis 0
# left stick vertical...... axis 1
# left trigger............. axis 2
# right stick horizontal... axis 3
# right stick vertical..... axis 4
# right trigger............ axis 5
# D-pad horizontal......... axis 6
# D-pad vertical........... axis 7